src/main/webapp/assets/js/function.js

```diff
@@ -0,0 +1,20 @@
+function loadFaqComponent() {
+    // Listener for FAQ
+    $('.panel').click(
+        function () {
+            if($(this).hasClass("collapsed")){
+                const collapseSign = $('.plusMinusSign');
```

Comment:
`.plusMinusSign` a bad CSS naming. Should be  `.plus-minus-sign` could you please fix it?

Reply:
Or it's more meaningful if we could rename it as `.collapse-sign-icon`.

What do you think?